TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/TeleOpMovement.kt

```diff
@@ -43,10 +44,26 @@ object TeleOpMovement : Component {
         var joyMagnitude = sqrt(joyY * joyY + joyX * joyX)
 
         // if the right bumper is pressed move slower
```

Comment:
Could you update this comment to reflect the change to use the `y` button?

Reply:
done